cmd/suffuse/main.go

```diff
@@ -1,11 +1,18 @@
 package main
 
-import . "github.com/suffuse/go-suffuse/suffuse"
+import (
+  . "github.com/suffuse/go-suffuse/suffuse"
+  "os"
+)
 
 func main() {
-  opts := ParseSfsOpts()
+  opts, opts_err := ParseSfsOpts(os.Args)
```

Comment:
It wouldn't work because we use the `PrintUsage` method. The `NewSfs` method returns a more generic `error`